packages/flutter_tools/lib/src/windows/build_windows.dart

```diff
@@ -92,6 +93,21 @@ Future<void> buildWindows(WindowsProject windowsProject, BuildInfo buildInfo, {
   } finally {
     status.stop();
   }
+
+  final String? binaryName = getCmakeExecutableName(windowsProject);
+  final File appFile = buildDirectory
+    .childDirectory('runner')
+    .childDirectory(sentenceCase(buildModeName))
+    .childFile('$binaryName.exe');
```

Comment:
This logic mirrors `BuildableWindowsApp.executable`: https://github.com/flutter/flutter/blob/61a2f9de51007999c8a3f50e98061d7138a694eb/packages/flutter_tools/lib/src/windows/application_package.dart#L112-L118

Please let me know if there's a better way to do this!